attributes/default.rb

```diff
@@ -179,7 +179,7 @@
 
 # Configure Abiquo websockify
 default['abiquo']['websockify']['port'] = 41338
-default['abiquo']['websockify']['address'] = '127.0.0.1'
+default['abiquo']['websockify']['address'] = '0.0.0.0'
 default['abiquo']['websockify']['api_url'] = 'https://localhost/api'
```

Comment:
Good! Then I'll add a change to remove it in this PR. Any ohther legacy WS config we should remove?